### ISSUE TITLE
Accessible Hacking

### ISSUE THREAD

**Issue by derekr** [jsfest/sf-cfp#2]
Or how to make your site/data as easy as possible to hack on with a goal of allowing almost anyone to contribute.
- NPM conventions (`npm start`, `npm test`) for all projects
- Using browserify and simple functions for frontend architecture
- Modular UI components
- Leveraging CORS to allow client side remote API hacking

**Comment by mikeal** [comment]
This would be great. @maxogden covered similar ideas in his talk the other day at RealtimeConf.

What I think might make it great is to show off a few projects/libraries that used these conventions and saw growth in usage, contributions and community. voxel.js might be one of those examples, or the new NodeBase ecosystem.

**Comment by max-mapper** [comment]
for reference: http://maxogden.github.io/slides/realtimeconf/#/8

**Comment by derekr** [comment]
I can't speak for open projects as much, but I'd probably reference a recent refactor here at diy.org where a lot of the items listed above were the focus.

**Comment by mikeal** [comment]
that would be great.

**Comment by derekr** [comment]
Good ref: http://www.nearform.com/nodecrunch/?p=109#.Un5kb5TXRWI on paypal simplifying their dev environment while moving to node

**Comment by techwraith** [comment]
Would you be interested in doing this talk at RejectJS on March 8th? We'd love to have you there :)

**Comment by derekr** [comment]
@Techwraith yeah I might be able to swing that! Mind sending some more info to drk@diy.org?
